pdfgeneration/pdfgeneration.py

```diff
@@ -154,52 +154,57 @@ def run(self, options):
         print(Gstr_title)
         print('Version: %s' % self.get_version())
         # fetch input data
-        with open('{}/prediction-default.json'.format(options.inputdir)) as f:
-          classification_data = json.load(f)
+        with open(f"{options.inputdir}/prediction-default.json") as f:
+            classification_data = json.load(f)
         try: 
-            with open('{}/severity.json'.format(options.inputdir)) as f:
+            with open(f"{options.inputdir}/severity.json") as f:
                 severityScores = json.load(f)
         except:
             severityScores = None
 
         # output pdf here
-        print("Creating pdf file in {}...".format(options.outputdir))
+        print(f"Creating pdf file in {options.outputdir}...")
         template_file = "pdf-covid-positive-template.html" 
         if classification_data['prediction'] != "COVID-19" or severityScores is None:
-          template_file = "pdf-covid-negative-template.html"
+            template_file = "pdf-covid-negative-template.html"
         # put image file in pdftemple folder to use it in pdf
         shutil.copy(options.inputdir + '/' + options.imagefile, "pdftemplate/")
-        with open("pdftemplate/{}".format(template_file)) as f:
+        with open(f"pdftemplate/{template_file}") as f:
             txt = f.read()
             # replace the values
             txt = txt.replace("${PATIENT_TOKEN}", options.patientId)
             txt = txt.replace("${PREDICTION_CLASSIFICATION}", classification_data['prediction'])
-            txt = txt.replace("${COVID-19}", classification_data['COVID-19'])
-            txt = txt.replace("${NORMAL}", classification_data['Normal'])
-            txt = txt.replace("${PNEUMONIA}", classification_data['Pneumonia'])
+            
+            prediction_analysis = ""
+            for columnName in classification_data:
+                prediction=classification_data.get(columnName, 'N/A')
+                if (columnName != 'prediction') and (columnName != 'Prediction') and (columnName != '**DISCLAIMER**'):
+                    prediction_analysis += f"<h3>{columnName}: <span>{prediction}</span></h3>"
```

Comment:
Variable names should be lowercase, with words separated by underscores as necessary to improve readability.

You can find guidelines and best practices of coding in Python in the
[PEP 8 documentations](https://www.python.org/dev/peps/pep-0008/).